### DISCUSSION TITLE
Make compatible with vs code themes

### DISCUSSION THREAD

**Discussion by benedikt-buchert** [dineug/erd-editor#283] [Q&A]
Is it possible to make the style adjust to the selected VS Code theme?

**Comment by benedikt-buchert** [comment]
https://github.com/vuerd/vuerd/issues/198

**Comment by dineug** [comment]
You can add the following settings to setting.json.
Because it's not real-time, it's reflected only when you load a new erd view.

```json
{
  "dineug.vuerd-vscode.themeSync": true,
  // will NOT synchronize the color list
  "dineug.vuerd-vscode.theme": {
    "keyPK": "#B4B400",
    "keyFK": "#dda8b1",
    "keyPFK": "#60b9c4",
    "edit": "#ffc107"
  }
}
```

**Comment by benedikt-buchert** [comment]
Thanks a lot for that hint.

**Comment by benedikt-buchert** [comment]
I think the only issue with the sync for me is that the canvas color is the same as the background color for the tables.

**Comment by dineug** [comment]
Most cases apply appropriately.
Some themes can't be helped.
<img width="563" alt="스크린샷 2021-12-20 오후 8 39 16" src="https://user-images.githubusercontent.com/45829489/146762242-dfca1b16-90c9-4778-9492-a4f092b6587b.png">
<img width="591" alt="스크린샷 2021-12-20 오후 8 39 37" src="https://user-images.githubusercontent.com/45829489/146762249-cf26cc10-9a2d-4e7f-a272-09136599e6be.png">
<img width="570" alt="스크린샷 2021-12-20 오후 8 39 49" src="https://user-images.githubusercontent.com/45829489/146762251-a1678aae-ae86-48d7-a95f-a01b82f2fa55.png">
<img width="530" alt="스크린샷 2021-12-20 오후 8 40 00" src="https://user-images.githubusercontent.com/45829489/146762256-b38b3472-ee5e-4f6a-8d60-6fa1ab7c2ed6.png">
<img width="551" alt="스크린샷 2021-12-20 오후 8 40 27" src="https://user-images.githubusercontent.com/45829489/146762260-898f10b0-f746-4352-a5b8-44fca1c677f3.png">
<img width="574" alt="스크린샷 2021-12-20 오후 8 40 41" src="https://user-images.githubusercontent.com/45829489/146762262-9a756fbb-bc86-45c7-8d4f-4627e84366b7.png">
<img width="488" alt="스크린샷 2021-12-20 오후 8 40 54" src="https://user-images.githubusercontent.com/45829489/146762264-2a998d28-3adc-4b72-934d-09b4c5dd786a.png">
<img width="580" alt="스크린샷 2021-12-20 오후 8 41 18" src="https://user-images.githubusercontent.com/45829489/146762265-61aa4e88-62e1-4445-aca9-2fa38f50c348.png">


Custom themes will need to be defined.
Open the webview developer tool
Find the vuerd-provider element
I'll look at the theme colors that are currently applied.
Define the customized theme accordingly.
- https://github.com/vuerd/vuerd/tree/master/packages/vuerd-vscode#example-settingsjson-configuration-for-a-light-themed-editor

<img width="546" alt="스크린샷 2021-12-20 오후 8 47 16" src="https://user-images.githubusercontent.com/45829489/146762732-60c11ad7-9ff8-4540-bab4-dd20cb0ac22c.png">
